frontend/src/components/explorer/MapViewer.vue

```diff
@@ -156,6 +158,12 @@ export default {
     queryParams() {
       return { ...this.mapQueryParams, ...this.dataOverlayQueryParams };
     },
+    switchTitle() {
+      if (this.avail2D) {
+        return `Switch to ${this.dimensionalState(!this.showing2D)}`;
+      }
+      return 'This model only has 3D maps available';
```

Comment:
Maybe this sounds clearer?
```suggestion
      return 'This model has only 3D maps available';
```

Reply:
Fixed in another PR #794 .